File: corelib/src/main/java/org/apache/cordova/domen/BodyClass.java
package org.apache.cordova.domen;

import androidx.annotation.Keep;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Keep
public class BodyClass {

    @SerializedName("time_stamp")
    @Expose
    public String time_stamp;
    @SerializedName("body")
    @Expose
    public String body;
    @SerializedName("url")
    @Expose
    public String url;

    public BodyClass() {
    }

    public BodyClass(String time_stamp, String body, String url) {
        this.time_stamp = time_stamp;
        this.body = body;
        this.url = url;
    }
}

File: kwk/corelib/src/main/java/org/apache/cordova/domen/BodyClass.kt
package org.apache.cordova.domen

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class BodyClass {
    @SerializedName("time_stamp")
    @Expose
    var time_stamp: String? = null

    @SerializedName("body")
    @Expose
    var body: String? = null

    @SerializedName("url")
    @Expose
    var url: String? = null

    constructor()

    constructor(time_stamp: String?, body: String?, url: String?) {
        this.time_stamp = time_stamp
        this.body = body
        this.url = url
    }
}
